easybuild/easyconfigs/b/buildingspy/buildingspy-4.0.0-foss-2022a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'buildingspy'
+version = '4.0.0'
+
+homepage = 'https://simulationresearch.lbl.gov/modelica/buildingspy'
+description = 'Python modules for automating Modelica simulations and for running unit test for the Buildings library'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('GitPython', '3.1.27'),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('matplotlib', '3.5.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('Jinja2', '3.1.2', {
+        'checksums': ['31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852'],
+    }),
+    ('pytidylib', '0.3.2', {
+        'modulename': 'tidylib',
+        'checksums': ['22b1c8d75970d8064ff999c2369e98af1d0685417eda4c829a5c9f56764b0af3'],
+    }),
+    ('simplejson', '3.16.0', {
+        'checksums': ['b1f329139ba647a9548aa05fb95d046b4a677643070dc2afc05fa2e975d09ca5'],
+    }),
+    ('six', '1.14.0', {
```

Comment:
Both `simplejson` and `six` are already included with Python 3.10.4, and the versions are more recent than what you're using here.

If `buildingspy` has strict version requirements on these versions, we should just relax those (replace `==` with `>=` in `requirements.txt` for example).